docs/reference/objects/package/step.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Package Step
+parent: Package
+---
+
+# Package Step
+{: .d-inline-block }
+object
+{: .label .fs-1 }
+
+Represents a creator-defined step along a [package]({% link docs/reference/objects/package.md %}) booking journey.
+
+<br>
+
+#### Attributes
+
+## `package_step.id`
+{: .d-inline-block }
+string
+{: .label .fs-1 }
+The database id of the step.
+
+## `package_step.name`
+{: .d-inline-block }
+string
+{: .label .fs-1 }
+The name given to the step.
+
+## `package_step.tagline`
+{: .d-inline-block }
+string
+{: .label .fs-1 }
+
+The description given to the step.
+
+## `package_step.product_groups`
+
+{: .d-inline-block }
+Array of packaged products
+{: .label .fs-1 }
+
+Each product containing variants added to the Package Step by the creator. Use `.variants` to get the variants added to the step for this product.
```

Comment:
In the package template both `product` and `variant` are called on the `product_group` - should there be another page breaking down what is available via the `product_group` drop? Or is the `product_group` just a `product` drop and we could remove the `.product` when passing through the product in `item.product` below?
```
{% for item in package_step.product_groups %}
    ...
          item: item.product,
          variants: item.variants, 
    ...
{% endfor %}
```

Reply:
There should; it's a "step product drop" - adding :+1: